splunk_eventgen/__main__.py

```diff
@@ -6,71 +6,128 @@
+    build_subparser.add_argument(
+        "--remove", default=True, help="Remove the build directory after completion.  Defaults to True")
     # WSGI subparser
     wsgi_subparser = subparsers.add_parser('wsgi', help="start a wsgi server to interact with eventgen.")
-    wsgi_subparser.add_argument("--daemon", action="store_true", help="Daemon will tell the wsgi server to start in a daemon mode and will release the cli.")
+    wsgi_subparser.add_argument(
+        "--daemon",
+        action="store_true",
+        help="Daemon will tell the wsgi server to start in a daemon mode and will release the cli.")
     # Service subparser
-    service_subparser = subparsers.add_parser('service', help="Run Eventgen as a Nameko service. Parameters for starting this service can be defined as either environment variables or CLI arguments, where environment variables takes precedence. See help for more info.")
-    service_subparser.add_argument("--role", "-r", type=str, default=None, required=True, choices=["controller", "server"], help="Define the role for this Eventgen node. Options: master, slave")
-    service_subparser.add_argument("--amqp-uri", type=str, default=None, help="Full URI to AMQP endpoint in the format pyamqp://<user>:<password>@<host>:<port>. This can also be set using the environment variable EVENTGEN_AMQP_URI")
-    service_subparser.add_argument("--amqp-host", type=str, default=None, help="Specify AMQP hostname. This can also be set using the environment variable EVENTGEN_AMQP_HOST. Default is localhost")
-    service_subparser.add_argument("--amqp-port", type=int, default=None, help="Specify AMQP port. This can also be set using the environment variable EVENTGEN_AMQP_PORT. Default is 5672")
-    service_subparser.add_argument("--amqp-webport", type=int, default=None, help="Specify AMQP web port. This can also be set using the environment variable EVENTGEN_AMQP_WEBPORT. Default is 15672")
-    service_subparser.add_argument("--amqp-user", type=str, default=None, help="Specify AMQP user. This can also be set using the environment variable EVENTGEN_AMQP_USER. Default is 'guest'")
-    service_subparser.add_argument("--amqp-pass", type=str, default=None, help="Specify AMQP password. This can also be set using the environment variable EVENTGEN_AMQP_PASS. Default is 'guest'")
-    service_subparser.add_argument("--web-server-address", type=str, default=None, help="Specify nameko webserver address. This can also be set using the environment variable EVENTGEN_WEB_SERVER_ADDR. Default is 0.0.0.0:9500")
+    service_subparser = subparsers.add_parser(
+        'service',
+        help="Run Eventgen as a Nameko service. Parameters for starting this service can be defined as either env" +
+             "variables or CLI arguments, where env variables takes precedence. See help for more info.")
+    service_subparser.add_argument(
```

Comment:
linter messed it up here :( I actually like the original code. this is harder to read.

Reply:
I disagree for this first 'service' parser because the help is much more readable.
However, I don't think it's necessary to break up every param as its own line (happens more often below this parser).